components/FlashCard.js

```diff
@@ -53,6 +94,23 @@ export default function FlashCard({ card, onMarkCorrect, collections }) {
             collectionTitle={collectionTitle}
           />
         )}
+        <StyledDeleteContainer>
+          {!isClicked ? (
+            <Button
+              $buttonVariant="delete"
+              name="Delete"
+              onClick={handleToggleButton}
+            />
```

Comment:
Instead of passing a `name` prop, you could use `children` so the button text is defined inside the component.

```suggestion
            <Button
              buttonVariant="delete"
              onClick={handleToggleButton} >Delete</Button>
```

You can apply the same to the other `Button` variants. Don't forget to adjust your `Button` component accordingly:

```js
export default function Button({ onClick, children, buttonVariant }) {
  return (
    <StyledButton $buttonVariant={buttonVariant} onClick={onClick}>
      {children}
    </StyledButton>
  );
}
```